content/en/docs/concepts/services-networking/service-topology.md

```diff
@@ -15,7 +15,15 @@ weight: 10
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
+
+{{< note >}}
+
+Service Topology is deprecated as of Kubernetes v1.21, and will be removed in
+v1.22. Topology Aware Hints was introduced in Kubernetes v1.21 and provides
```

Comment:
```suggestion
Service Topology is deprecated as of Kubernetes v1.21.
Topology Aware Hints was introduced in Kubernetes v1.21 and provides
```
Small nit, typically the docs do not make future statements.

Reply:
Updated, thanks!